pages/gallery.html

```diff
@@ -1,12 +1,46 @@
 <!DOCTYPE html>
 <html lang="en">
-<head>
-  <meta charset="UTF-8">
-  <meta http-equiv="X-UA-Compatible" content="IE=edge">
-  <meta name="viewport" content="width=device-width, initial-scale=1.0">
-  <title>Document</title>
-</head>
-<body>
-  
-</body>
-</html>
\ No newline at end of file
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Document</title>
+    <link rel="stylesheet"  href="../styles/stylesheet.css" />
+    <link rel="stylesheet"  href="../styles/gallery.css" />
+  </head>
+  <body>
+    <header>
+      <h1>Shark Appreciation Page</h1>
+    </header>
+    <img
+      src="../assets/reefsharkwikipedia.jpg"
+      alt="reef shark swimming in the open sea"
+    />
+    <img
+      src= "../assets/Mauricio-Hoyos-5.jpg"
+      alt="shark 2"
+    />
+    <img
+      src= "../assets/lemon-sharks-1.jpg.avif"
+      alt = "shark 3"
+    />
+    <img
+    src= "../assets/_20230323_on_whale_sharks.jpg"
+    alt = "shark 4"
+    />
+    <img
+    src= "../assets/istockphoto-454433897-612x612.jpg"
+    alt = "shark 5"
+    />
+    <img
+    src= "../assets/sharks-orig.jpg"
```

Comment:
Some indentation is 2-spaces and some is 4-spaces, to make things easy to read and reduce possible confusion, we should pick one and be consistent across the page.